jenkins_wrapper_cookbook/nodes/.gitignore

```diff
@@ -0,0 +1,2 @@
+# This file exists to allow checking in the empty directory required by Chef-zero
```

Comment:
This is actually to fix the Vagrant chef-zero provisioner, as it requires a `nodes_path` and fails to start if the directory doesn't exist.